docs/_docs/integration-checks-reference.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Integration Checks Reference
-order: 22
+order: 24
```

Comment:
This is shifted two places because we have an existing but currently excluded doc at order 23.